elytron/src/main/java/org/wildfly/extension/elytron/CredentialStoreService.java

```diff
@@ -114,6 +114,9 @@ public void start(StartContext startContext) throws StartException {
             credentialStoreAttributes.put(CS_LOCATION_ATTRIBUTE, loc.toAbsolutePath().toString());
             credentialStore = getCredentialStoreInstance();
             credentialStore.initialize(credentialStoreAttributes, resolveCredentialStoreProtectionParameter(), otherProviders.getOptionalValue());
+            if ( credentialStoreAttributes.get(ElytronDescriptionConstants.CREATE).equals("true") && !loc.toFile().exists() ){
+                credentialStore.flush();
```

Comment:
@pskopek could you please have a look at the comment from Brian?  If something else is needed if we can have a PR against ladybird I will merge it into this PR.